api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/ClaimantTestDataFactory.java

```diff
@@ -54,11 +52,7 @@ public static Claimant aValidClaimant() {
      * @return The built Claimant
      */
     public static Claimant aClaimantWithLastName(String lastName) {
-        return Claimant.builder()
-                .firstName("Joe")
-                .lastName(lastName)
-                .nino(VALID_NINO)
-                .build();
+        return buildClaimant(VALID_FIRST_NAME, lastName, VALID_NINO);
```

Comment:
Not worth changing right now, but I'd suggest:
```
public static Claimant aClaimantWithLastName(String lastName) {
    return aValidClaimantBuilder().lastName(lastName).build;
}
```

Reply:
Yes, good suggestion, done.